tests/medium/test_swift.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2014 Canonical
+#
+# Authors:
+#  Didier Roche
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation; version 3.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+"""Tests for swift"""
+
+from . import ContainerTests
+import os
+from ..large import test_swift
+
+
+class SwiftInContainer(ContainerTests, test_swift.SwiftTests):
```

Comment:
Looking great, adding some invalid data info tests (like invalid download page, invalid gpg, invalid url) to tests those kinds of errors would be great!

You can have a look at what I added the other day to Eclipse medium tests: https://github.com/ubuntu/ubuntu-make/blob/master/tests/medium/test_ide.py#L45
(Note: we only do that when we override `BaseInstaller` methods, as otherwise, this is covered by `test_baseinstaller.py`)